Dockerfiles/mariadb/mariadb.conf

```diff
@@ -0,0 +1,17 @@
+[mariadb]
+# File Key Management
+plugin_load_add = file_key_management
+file_key_management_filename = /etc/mysql/encrypt/keys.enc
+file_key_management_filekey = FILE:/etc/mysql/encrypt/pass.key
+file_key_management_encryption_algorithm = AES_CTR
+
+# InnoDB/XtraDB Encryption Setup
+innodb_default_encryption_key_id = 1
+innodb_encrypt_tables = FORCE
+innodb_encrypt_temporary_tables = ON
+innodb_encrypt_log = ON
+innodb_encryption_threads = 4
+
+encrypt-tmp-disk-tables = 1
```

Comment:
According to documentation, this is written as `encrypt_tmp_disk_tables=ON`
[https://mariadb.com/kb/en/server-system-variables/#encrypt_tmp_disk_tables]
By the way there is an **if** in its description. Are we using aria for temp tables?

Reply:
A mix-up of CLI vs conf file

Reply:
We use Innodb, so not needed.